dequorum/templates/dequorum/thread_detail.html

```diff
@@ -4,21 +4,23 @@
 
 <h1> {{ thread.title }} [{{ messages.count }} messages] </h1>
 
-<ol>
 {% for message in messages %}
-    <li>
-        <h2> By: {{ message.author }} on {{ message.created|date }}</h2>
-        <div> {{ message.body }} </div>
-    </li>
+    <div class="messageBox">
+        <div class="{% cycle 'messageHead1' 'messageHead2' %}">
+            <h2> By: {{ message.author }} </h2>
+            <p class="dateTime">{{ message.created|date }} | {{ message.created|time }}</p>
```

Comment:
This should probably use the <time> tag...

    <time datetime="{{ message.created }}"> { pretty formatted date/time goes here}  </time>